client/packages/android/app/src/main/java/org/openmsupply/client/ExtendedWebViewClient.java

```diff
@@ -34,7 +34,7 @@ public void onPageStarted(WebView webView,
         // There is no way to get the full list of plugins from bridge, use 'debug' and
         // see what plugins to add
         List<PluginHandle> pluginList = Arrays.asList(bridge.getPlugin("NativeApi"), bridge.getPlugin("Keyboard"),
-                bridge.getPlugin("CertPlugin"), bridge.getPlugin("WebView"), bridge.getPlugin("BarcodeScanner"));
+                bridge.getPlugin("WebView"), bridge.getPlugin("BarcodeScanner"));
```

Comment:
Cert plugin was removed, CertWebViewClient is now set in NativeApi (since we need to pass connectedServer to CertWebViewClient)